feat(vpc): vpc module (#1)
scaleway/terraform-scaleway-vpc-module: README.md

> # terraform-scaleway-vpc-module

Reviewer: ```suggestion
# Scaleway VPC Terraform module
```

> This module can be used to deploy a VPC with various resources. Common deployment examples can be found in [examples/](./examples).

Reviewer: ```suggestion
Terraform module that can be used to deploy VPC resources on Scaleway. Common deployment examples can be found in [examples/](./examples).
```

## Usage

> The example below builds a simple VPC with a PGW, a Load Balancer and some instances behind it.

Reviewer: ```suggestion
The example below provision a basic VPC with a Public Gateway and a Load Balancer with some instances behind it.
```

# terraform-scaleway-vpc-module

> This module can be used to deploy a VPC with various resources. Common deployment examples can be found in [examples/]().

Reviewer 2: Broken link?
